portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+function addRandomLanguage() {
+  const languages =
+      ['Español', 'English', 'Deutsch'];
+ 
+  // Pick a random language.
+  const language = languages[Math.floor(Math.random() * languages.length)];
+ 
```

Comment:
Remove extra space (but you can keep the newline)

Reply:
Same as in the past review, I erased the extra space inside the array slot identifier (if you meant something different, I'll happily correct as deemed).